js/script.js

```diff
@@ -0,0 +1,63 @@
+var posts = {};
+var counter = 0;
+ 
+$("#stop").hide();
+
+$(document).ready(function() {
+
+	$("form").on("submit", function(event) {
+		event.preventDefault();
+		// console.log("button works");
+
+
+			var searchTerm = $("#search-box").val();
+			$.get("http://www.reddit.com/search.json?q=" + searchTerm + "+nsfw:no").done(function(data) {
+				posts = data;
+				// console.log(posts);
+
+				// clear search term and hide form after search button clicked;
+				$("form").hide();
+				$("input").val("");
+
+				// show stop button at start of slide show
+				$("#stop").show();
+
+				var searchResults = [];
+				for (i = 0; i < posts.data.children.length; i++) {
+					if (posts.data.children[i].data.thumbnail !== 'self') {
```

Comment:
The reason you were getting some blank images with alt 'image responsive' is because you were using the ```posts.data.children[i].data.thumbnail``` value instead of the ```posts.data.children[i].data.preview.images``` array. Since reddit posts may or may not have a thumbnail, you may get some blanks. I would do something like this instead:
```
if (posts.data.children[i].data.preview && posts.data.children[i].data.preview.images[0].source.url !== undefined) {
    searchResults.push(posts.data.children[i].data.preview.images[0].source.url);
} 
```